### PR TITLE
fix(Select): change dropdown fields style in the Settings window

### PR DESCRIPTION
**Pull request by BenjaminGolya**
PR for: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-271751

Dropdown style changes.

### CODE REVIEW

File: src/styles/_colors.scss [hunk 1]
```diff
@@ -26,6 +26,9 @@ $wxc-blue-90: #0A274A;
 $wxc-blue-95: #091A30;
 $wxc-blue-100: #07101C;
 
+$wxc-blue-40-alpha-30: rgba(100, 180, 250, .3);
```
Comment (lalli-flores) [inline src/styles/_colors.scss:29]: HEX values are used for all other colors, could you maintain consistency and convert this RGBA values?

File: src/data/meetings.js [hunk 2]
```diff
@@ -19,9 +19,9 @@ export default {
     state: MeetingState.JOINED,
     showRoster: false,
     showSettings: false,
-    cameraID: 'cameraID1',
```
Comment (lalli-flores) [inline src/data/meetings.js:22]: Why did you remove these values?

Comment (BenjaminGolya) [reply src/data/meetings.js:22]: For testing only, I wanted to simulate when we have nothing selected for cameraID, microphoneID and speakerID
I will reset to the old values

File: src/styles/_colors.scss (hunk 1 above)
Comment (BenjaminGolya) [reply src/styles/_colors.scss]: Right, I will convert

File: src/themes/light.scss [hunk 3]
```diff
@@ -117,19 +117,21 @@
...
 
   --wxc-options-list-item--active--background: #{$wxc-black-alpha-11};
 
   --wxc-options-list-item--focus--box-shadow: 0 0 0 1px #{$wxc-blue-60} inset;
 
-  --wxc-options-list-item--selected--color:  #{$wxc-blue-40};
-
   --wxc-scrollbar-thumb: #{$wxc-black-alpha-11};
+
+  --wxc-options-list-item--selected--color:  #{$wxc-blue-60};
```
Comment (cipak) [inline src/themes/light.scss:136]: this should go together with the other options-list variables

**Review by cipak — APPROVED** [review]
